static/index.html

```diff
@@ -42,7 +42,7 @@ <h1 class="title">Remote Play</h1>
       <div class="touchpad" id="touchpad">
         <p>
           Use this area as a touchpad.<br />Supported gestures:<br />track and
-          tap (Left Mouse Button).
+          tap (Left Mouse Button).<br />Double Tap (Right Mouse Button).
```

Comment:
Let's break this into multiple lines for readability. Also, we should have a line break before `Double tap`, as that's also a supported gesture:
```html
Use this area as a touchpad.<br />
Supported gestures:<br />
Track, tap (left mouse button), and double tap (right mouse button).
```